src/platforms/common/profiling/index.mdx

```diff
@@ -383,6 +428,17 @@ If you don't see any profiling data in [sentry.io](https://sentry.io), you can t
 - If the automatic instrumentation is not sending performance data, try using <PlatformLink to="/performance/instrumentation/custom-instrumentation">custom instrumentation</PlatformLink>.
 - Enable <PlatformLink to="/configuration/options/#debug">debug mode</PlatformLink> in the SDK and check the logs.
 
+<PlatformSection supported={["react-native"]}>
+
+### Limitations
+
+Profile samples are collected periodically and sampling frequency is fixed make sure to check whether the overhead is within the acceptable range for your use case.
+We are working on symbolication of the collected profiles. Out of the box, you will see function names.
+
+As always, and especially with Profiling in React Native being an alpha feature, feedback is welcome on [Discord](https://discord.com/channels/621778831602221064/750735628932612096) or [GitHub](https://github.com/getsentry/sentry-react-native/issues/2668).
```

Comment:
```suggestion
As always, and especially since Profiling in React Native is an alpha feature, feedback is welcome on [Discord](https://discord.com/channels/621778831602221064/750735628932612096) or [GitHub](https://github.com/getsentry/sentry-react-native/issues/2668).
```